report.typ

```diff
@@ -67,62 +21,105 @@
       }
     }
     acc
-  },
+  }
+    .sorted()
+    .map(name => if name == "Turnverein Pflanzschule" {
+      "Tv. Pflanz."
+    } else {
+      name
+    }),
```

Comment:
This is not necessary. This entry will never have an `attending` value set.

Reply:
removed